README.md

```diff
@@ -21,7 +21,9 @@ on:
   uses: devops-actions/action-get-tag@v1.1
   with:
     # Optionally strip `v` prefix
```

Comment:
Let's remove this useless comment now, since it is included in the next line

Reply:
Thanks for your review. You're right, fixed in https://github.com/devops-actions/action-get-tag/pull/28/commits/8c9c4fb35c7f2c4532b86ee22a58a0c0e41bf60c.